src/gauge/radial-gauge/radial-gauge.component.css

```diff
@@ -0,0 +1,11 @@
+/* .gauge-tick-large path {
+  stroke-width: 2px;
+}
+
+.gauge-tick-small path {
+  stroke-width: 1px;
+}
+
+.gauge-tick path {
+    stroke: #666;
+} */
```

Comment:
Did you intend to leave this file with these commented out css styles?

It looks to be the same CSS at:
`ngx-charts/src/gauge/gauge.component.scss`